src/seedu/addressbook/commands/HelpCommand.java

```diff
@@ -18,7 +18,8 @@ public CommandResult execute() {
                 + "\n" + DeleteCommand.MESSAGE_USAGE
                 + "\n" + ClearCommand.MESSAGE_USAGE
                 + "\n" + FindCommand.MESSAGE_USAGE
-                + "\n" + ListCommand.MESSAGE_USAGE
+                + "\n" + ListCommand.MESSAGE_USAGE 
+                + "\n" + SortCommand.MESSAGE_USAGE
```

Comment:
You didn't update the I/O tests to reflect this change.

Reply:
Good catch! I will update it to reflect this change!